builtin/k8s/helm/platform.go

```diff
@@ -117,12 +117,12 @@ func (p *Platform) Deploy(
 		client.NameTemplate = ""
 		client.OutputDir = ""
 		client.Atomic = false
-		client.SkipCRDs = false
+		client.SkipCRDs = p.config.SkipCRDs
```

Comment:
We should probably also make this configurable for _upgrading_ Helm charts. In the snippet below, it's hard-coded to false as well. Making this configurable here is only for the 1st installation of the chart.

https://github.com/hashicorp/waypoint/blob/ded15e17ddd6c2bcb896665efd6ec7de559d8508/builtin/k8s/helm/platform.go#L152

Reply:
Missed this part too, good point!